Source/SwiftLintBuiltInRules/Rules/Lint/TodoRule.swift

```diff
@@ -26,41 +26,55 @@ struct TodoRule: SwiftSyntaxRule, ConfigurationProviderRule {
     )
 
     func makeVisitor(file: SwiftLintFile) -> ViolationsSyntaxVisitor {
-        Visitor(viewMode: .sourceAccurate)
+        Visitor(only: configuration.only)
     }
 }
 
 private extension TodoRule {
     final class Visitor: ViolationsSyntaxVisitor {
+        private let only: [TodoKeyword]
+
+        init(only: [TodoKeyword]) {
+            self.only = only
+            super.init(viewMode: .sourceAccurate)
+        }
+
         override func visitPost(_ node: TokenSyntax) {
-            let leadingViolations = node.leadingTrivia.violations(offset: node.position)
-            let trailingViolations = node.trailingTrivia.violations(offset: node.endPositionBeforeTrailingTrivia)
+            let leadingViolations = node.leadingTrivia.violations(offset: node.position, only: only)
+            let trailingViolations = node.trailingTrivia.violations(offset: node.endPositionBeforeTrailingTrivia,
+                                                                    only: only)
             violations.append(contentsOf: leadingViolations + trailingViolations)
         }
     }
 }
 
 private extension Trivia {
-    func violations(offset: AbsolutePosition) -> [ReasonedRuleViolation] {
+    func violations(offset: AbsolutePosition, only: [TodoKeyword]) -> [ReasonedRuleViolation] {
         var position = offset
         var violations = [ReasonedRuleViolation]()
         for piece in self {
-            violations.append(contentsOf: piece.violations(offset: position))
+            violations.append(contentsOf: piece.violations(offset: position, only: only))
             position += piece.sourceLength
         }
         return violations
     }
 }
 
 private extension TriviaPiece {
-    func violations(offset: AbsolutePosition) -> [ReasonedRuleViolation] {
+    func violations(offset: AbsolutePosition, only: [TodoKeyword]) -> [ReasonedRuleViolation] {
```

Comment:
`only` seems a bit short. I'd like a more descriptive name like `todoKeywords` more:
```suggestion
    func violations(offset: AbsolutePosition, for todoKeywords: [TodoKeyword]) -> [ReasonedRuleViolation] {
```